operator/pkg/apis/istio/v1alpha1/validation/validation.go

```diff
@@ -107,6 +107,8 @@ func deprecatedSettingsMessage(iop *v1alpha1.IstioOperatorSpec) string {
 		{"Values.global.tracer.stackdriver.maxNumberOfAnnotations", "meshConfig.defaultConfig.tracing.stackdriver.maxNumberOfAnnotations", 0},
 		{"Values.global.tracer.stackdriver.maxNumberOfMessageEvents", "meshConfig.defaultConfig.tracing.stackdriver.maxNumberOfMessageEvents", 0},
 		{"Values.global.tracer.datadog.address", "meshConfig.defaultConfig.tracing.datadog.address", ""},
+		{"Values.global.meshExpansion.enabled", "Gateway and other Istio networking resources, such as in samples/istiod-gateway/", false},
+		{"Values.gateways.istio-ingressgateway.meshExpansionPorts", "components.ingressGateways[name=istio-ingressgateway].k8s.service.ports", nil},
```

Comment:
nit: I don't think a warning will help the user much, because I suspect they are not trying to customize the ports but add the normal ones (15012/15017). May be better to just point to the samples as well?

Reply:
I thought that if they were only using the normal ones, they wouldn't be overriding the value so it would just use the default, so this message wouldn't appear

Reply:
do we have the corresponding doc PR to remove the usage of these two configs?

Reply:
https://github.com/istio/istio.io/pull/7869